layouts/partials/site_footer.html

```diff
@@ -2,20 +2,21 @@
   {{ if or (site.GetPage "terms.md") (site.GetPage "privacy.md") }}
   <p class="powered-by">
     {{ with site.GetPage "privacy.md" }}
-      {{ printf "<a href=\"%s\">%s</a>" .RelPermalink .Title | safeHTML }}
```

Comment:
Please revert all the changes to this file as per previous comment - this needs to be considered separately

Reply:
Git is still showing that changes are present in this file...